src/java.base/share/classes/java/util/zip/DeflaterInputStream.java

```diff
@@ -34,6 +34,20 @@
  * Implements an input stream filter for compressing data in the "deflate"
  * compression format.
  *
+ * <h2 id="compressor-usage">Compressor Usage</h2>
+ * A {@linkplain DeflaterInputStream} created without
```

Comment:
I don't think you meant to use @linkplain here, should be @code or @link.

Reply:
That was meant to be `@code`. Fixed now.